bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -515,11 +515,11 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.45.0
+  replaces: jaeger-operator.v1.46.0
```

Comment:
https://github.com/jaegertracing/jaeger-operator/actions/runs/5534600439/jobs/10099768942?pr=2263#step:6:279

The CI expects 1.47.0 for some reason.

Reply:
and
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/jaegertracing_jaeger-operator/2263/pull-ci-jaegertracing-jaeger-operator-main-jaeger-e2e-tests/1679186532214247424#1:build-log.txt%3A22